Базовая стилизация всех страниц

ConfidentPie opened the pull request:
---
:mortar_board: [Базовая стилизация всех страниц](https://up.htmlacademy.ru/htmlcss/35/user/2216859/tasks/8)

ConfidentPie committed:
Подключил фоны, шрифты

senchkim commented on styles/styles.css:
```diff
@@ -0,0 +1,255 @@
+@font-face {
```
#Совет
Загляни в папку `images` - там еще есть картинки названные с большими буквами, 
а также вот такое - `greenTreeInn.jpg` => `green-tree-inn.jpg` - в качестве разделителя используй тире

senchkim commented on styles/styles.css:
```diff
@@ -0,0 +1,255 @@
+}
+
+.navigation-link {
+    font-family: 'PT Sans' sans-serif;
```
`font-family: 'PT Sans' sans-serif;`
Ошибка массовая, забыл запятую указать! смотри строку 18 - там все верно

senchkim commented on styles/styles.css:
```diff
@@ -0,0 +1,255 @@
+}
+
+body {
+font-family: "PT Sans", sans-serif;
```
тут кажется не хватает форматирования

senchkim commented on styles/styles.css:
```diff
@@ -0,0 +1,255 @@
+    font-weight: 700;
+    font-size: 20px;
+    line-height: 24px;
+    transform: capitalize;
```
https://developer.mozilla.org/ru/docs/Web/CSS/transform
ошибка, нет такого значения у этого свойства!

senchkim commented on styles/styles.css:
```diff
@@ -0,0 +1,255 @@
+
+.product-cards-link {
+    color: #000000;
+    font-family: 'PT Sans';
```
Забыл указать семейство шрифта здесь и ниже по коду тоже!

senchkim commented on styles/styles.css:
```diff
@@ -0,0 +1,255 @@
+}
+
+.newsletter-subscription {
+color: #FFFFFF;
```
Форматирование поехало

ConfidentPie replied:
Не понял что поехало?(

senchkim commented on styles/styles.css:
```diff
@@ -0,0 +1,255 @@
+    font-weight: 400;
+    font-size: 22px;
+    line-height: 26px;
+    color:#333333;
```
на строке 139 значение у цвета с большой буквы, а тут с маленькой - надо сделать одинаково!

ConfidentPie replied:
Если я все цвета пропишу с маленькой буквы, это не будет ошибкой? Или лучше все с большой? VSCode оба варианта допускает. В figma  же все с большой буквы.

ConfidentPie replied:
Сделал с маленькой буквы все цвета

ConfidentPie committed:
Исправил указанные ошибки

1.Переименовал и поправил в коде названия отелей.
2. Проверил другие названия изображений, привел к единому виду.
3. Исправил опечатке в указании стилей шрифтов, расставил запятые

ConfidentPie committed:
Поправил hero исходя из гипотезы - что фон и оформление - это декоративные элементы(записал в CSS), а текст привествия

ConfidentPie committed:
Сделал отступы в селекторах
